### PR TITLE
Adding queue pulling and KairosDB handling

### PR DESCRIPTION
**Pull request by ch40s3k**
Added:

ZeroMQ pulling,
KairosDB plugin

### CODE REVIEW

File: alamo_worker/plugins/kairosdb.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import requests
+from requests.exceptions import ConnectionError, HTTPError
+import simplejson
```
Comment (pnuckowski) [inline alamo_worker/plugins/kairosdb.py]: import order?

File: alamo_worker/plugins/kairosdb.py [hunk 2]
```diff
@@ -11,6 +15,27 @@
...
+    _endpoint = '/api/v1/datapoints/query'
 
     def execute(self, check):
-        logger.info('KairosDB: {}'.format(check))
+        endpoint = "http://{}:{}{}".format(self._cfg['host'],
+                                           self._cfg['port'],
+                                           self._endpoint)
+
+        try:
+
```
Comment (pnuckowski) [inline alamo_worker/plugins/kairosdb.py]: blank line.

File: alamo_worker/plugins/kairosdb.py [hunk 3]
```diff
@@ -11,6 +15,27 @@
...
+            response = requests.post(endpoint,
+                                     data=simplejson.dumps(check['query']))
```
Comment (deejay1) [inline alamo_worker/plugins/kairosdb.py]: why simplejson instead of json from the standard library?

File: alamo_worker/queue.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import zmq
```
Comment (deejay1) [inline alamo_worker/queue.py]: not aiomq?

File: vagrant/provision.sh [hunk 5]
```diff
@@ -29,4 +29,4 @@ fi
 
 # Install application environment
 su -l vagrant -c '/home/vagrant/.virtualenvs/alamo/bin/pip install ipython tox'
-su -l vagrant -c '/home/vagrant/.virtualenvs/alamo/bin/pip install asyncio asynqp'
+su -l vagrant -c '/home/vagrant/.virtualenvs/alamo/bin/pip install asyncio zmq requests simplejson'
```
Comment (deejay1) [inline vagrant/provision.sh]: aren't we using python3.4? there's no need to install and outside asyncio

**Pull request closed without merging**